resources/views/board.blade.php

```diff
@@ -0,0 +1,53 @@
+@extends('layouts.app')
```

Comment:
Wasn't this file supposed to be part of another task?

Reply:
Yes, sorry, I reverted the commits.